tests/api_app/test_config_parse.py

```diff
@@ -0,0 +1,77 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+import random
+from hashlib import md5
+
+from django.test import TestCase
+
+from api_app.core.serializers import AbstractConfigSerializer
+
+
+class ModAbstractConfigSerializer(AbstractConfigSerializer):
+    input_config = None
+
+    @classmethod
+    def _md5_config_file(cls) -> str:
+        return md5(str(random.random()).encode("utf-8")).hexdigest()
+
+    @classmethod
+    def _read_config(cls):
+        return cls.input_config
```

Comment:
This is a smart way to mock the function, but I prefer the use of the unittest library. You can use the `patch` decorator to mock the `_read_config` return.

Reply:
Done